icons/mail-dot-ru.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Mail.Ru icon</title><path d="M11.585 5.267c1.834 0 3.558.811 4.824 2.08v.004c0-.609.41-1.068.979-1.068h.145c.891 0 1.073.842 1.073 1.109l.005 9.475c-.063.621.64.941 1.029.543 1.521-1.564 3.342-8.038-.946-11.79-3.996-3.497-9.357-2.921-12.209-.955-3.031 2.091-4.971 6.718-3.086 11.064 2.054 4.74 7.931 6.152 11.424 4.744 1.769-.715 2.586 1.676.749 2.457-2.776 1.184-10.502 1.064-14.11-5.188C-.977 13.521-.847 6.093 5.62 2.245 10.567-.698 17.09.117 21.022 4.224c4.111 4.294 3.872 12.334-.139 15.461-1.816 1.42-4.516.037-4.498-2.031l-.019-.678c-1.265 1.256-2.948 1.988-4.782 1.988-3.625 0-6.813-3.189-6.813-6.812 0-3.659 3.189-6.885 6.814-6.885zm4.561 6.623c-.137-2.653-2.106-4.249-4.484-4.249h-.09c-2.745 0-4.268 2.159-4.268 4.61 0 2.747 1.842 4.481 4.256 4.481 2.693 0 4.464-1.973 4.592-4.306l-.006-.536z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Mail.Ru icon</title><path d="M11.5848 5.2664c1.834 0 3.558.811 4.824 2.08v.004c0-.609.4101-1.068.9791-1.068h.145c.891 0 1.073.842 1.073 1.109l.005 9.4753c-.063.621.64.941 1.029.543 1.5211-1.564 3.3422-8.0382-.946-11.7903-3.996-3.4971-9.3572-2.921-12.2093-.955-3.031 2.091-4.9711 6.7182-3.086 11.0643 2.054 4.7401 7.9311 6.1521 11.4242 4.7441 1.7691-.715 2.5861 1.676.749 2.457-2.776 1.1841-10.5022 1.0641-14.1103-5.188-2.439-4.2212-2.309-11.6494 4.158-15.4975C10.5668-.6987 17.09.1163 21.022 4.2233c4.1111 4.2942 3.8721 12.3344-.139 15.4615-1.816 1.42-4.5161.037-4.4981-2.031l-.019-.678c-1.265 1.256-2.948 1.988-4.7821 1.988-3.6251 0-6.8132-3.1891-6.8132-6.8122 0-3.6591 3.189-6.8852 6.8142-6.8852zm4.561 6.6232c-.137-2.6531-2.106-4.2491-4.484-4.2491h-.09c-2.7451 0-4.2682 2.159-4.2682 4.61 0 2.7472 1.842 4.4812 4.2562 4.4812 2.693 0 4.464-1.973 4.592-4.3061z"/></svg>
```

Comment:
Looks like their logo has changed! 😅 Could you look to rebuild from source, with a precision of 3 please? 🙏🏻
![image](https://user-images.githubusercontent.com/1830867/103037279-62681800-4563-11eb-8d5c-2cbc082cf2c3.png)

Reply:
@adamrusted I think the source doesn't really fit here. The source is for the "mail.ru group", the company behind "mail.ru". Altough they do use the same "@" as logo. But for example the colors are different.

I've now updated the source to https://my.mail.ru instead, where a SVG can be found in the header. I've as well updated the color to orange `#FF9E00` (found in the SVG in the header). Do you agree?

![mail-dot-ru](https://user-images.githubusercontent.com/70264417/103139519-5dc76f00-46dd-11eb-9cee-b7071c2220d6.png)

Reply:
Hmm. While the orange does seem to be the colour of the `@`, looking at their social media channels it would appear the blue (albeit a darker `#005FF9`) is still their main brand colour. Could another @simple-icons/maintainers weigh in on this before we commit please?

Reply:
I would opt by `#005FF9` too, seems to be more representative taking into account other subdomains. Mail.ru Group uses  `#FC2C38` in [their corportative subdomain](https://corp.mail.ru/ru/), although the same logo.

Reply:
Thanks for the feedback. The color is `#005FF9` now. 

![mail-dot-ru](https://user-images.githubusercontent.com/70264417/103175638-ba589480-486b-11eb-85c2-0164a09b92cf.png)